Uni_life_tracker/app/src/main/java/com/generic/plannr/Gateways/EventGatewayInterface.java

```diff
@@ -0,0 +1,15 @@
+package com.generic.plannr.Gateways;
+
+import com.generic.plannr.Entities.Event;
+import com.generic.plannr.Entities.User;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public interface EventGatewayInterface {
```

Comment:
Don't forget javadocs here for the class and the methods (even if there is no implementation technically)

Reply:
Done